model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,98 @@
+    instrument: updowncounter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.queue_capacity
+    type: metric
+    metric_name: telemetry.sdk.span.processor.queue_capacity
+    stability: experimental
+    brief: "The maximum number of spans the queue of a given instance of an SDK span processor can hold"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: gauge
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.spans_submitted
+    type: metric
+    metric_name: telemetry.sdk.span.processor.spans_submitted
+    stability: experimental
+    brief: "The number of spans submitted for processing to this span processor."
+    note: |
+      A span is considered submitted when the first callback has been called which triggers processing of the span for the given processor.
+      For the Batching and Simple Span Processors this metric therefore MUST be incremented on invocations of the `on_end` span processor callback.
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.spans_processed
+    type: metric
+    metric_name: telemetry.sdk.span.processor.spans_processed
+    stability: experimental
+    brief: "The number of spans for which the processing has finished, either successful or failed"
+    note: |
+      For successful processing, `error.type` must be empty. For failed processing, `error.type` must contain the failure cause.
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+      - ref: error.type
+        brief: >
+          A low-cardinality description of the failure reason. SDK Batching Span Processors MUST use `queue full` for spans dropped due to a full queue.
+        examples: ["queue full"]
+
+  - id: metric.telemetry.sdk.span.exporter.spans_submitted
+    type: metric
+    metric_name: telemetry.sdk.span.exporter.spans_submitted
+    stability: experimental
+    brief: "The number of spans for which the `export` callback of the exporter has been invoked. This does not require the export of the span to have finished yet."
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.exporter.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.exporter.spans_processed
+    type: metric
+    metric_name: telemetry.sdk.span.exporter.spans_processed
+    stability: experimental
+    brief: "The number of spans for which the export has finished, either successful or failed"
+    note: |
```

Comment:
I would like to see an explicit specification for how the SDK handles the "spans_rejected" count returned on successful exports. My preference is to translate these into failure counts w/ error.type = `rejected`.

Reply:
Good point, added in f1e1404bf12892